ament_index_python/ament_index_python/cli.py

```diff
@@ -14,13 +14,13 @@
 
 import argparse
 import sys
+from typing import Any, Dict, Generator, List, Tuple, Optional, Union
 
-from ament_index_python.resources import get_resource
-from ament_index_python.resources import get_resource_types
-from ament_index_python.resources import get_resources
+from ament_index_python.resources import (get_resource, get_resource_types,
+                                          get_resources)
```

Comment:
Any particular reason for this change?  Given that we have to split it on multiple lines anyway, and it isn't directly related to what you are trying to accomplish here, I'll suggest just leaving this as-is.